android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -997,6 +1004,27 @@ private void initializePlayerSource(Source runningSource) {
         finishPlayerInitialization();
     }
 
+    private DrmSessionManager initializePlayerDrm() {
+        DrmSessionManager drmSessionManager = null;
+        DRMProps drmProps = source.getDrmProps();
+        // need to realign UUID in DRM Props from source
+        if (drmProps != null && drmProps.getDrmType() != null) {
+            UUID uuid = Util.getDrmUuid(drmProps.getDrmType());
+            if (uuid != null) {
+                try {
+                    DebugLog.w(TAG, "drm buildDrmSessionManager");
```

Comment:
Not your code, but I think we should remove this warning (which is not a warning in fact)

```suggestion
                    DebugLog.d(TAG, "drm buildDrmSessionManager");
```

Reply:
Done ✅